src/patternfly/demos/PageLayout/docs/design.md

```diff
@@ -0,0 +1,34 @@
+# Component Name
```

Comment:
Fill in this file with at least a description of the demo.

Reply:
Updated